apps/web/src/services/iam/components/UserGroupTabDetail.vue

```diff
@@ -0,0 +1,175 @@
+        visible: USER_GROUP_MODAL_TYPE.EDIT,
+        themeColor: 'primary',
+    }); break;
+    case USER_GROUP_MODAL_TYPE.STATUS: userGroupPageStore.updateModalSettings({
+        type,
+        title: i18n.t('IAM.USERGROUP.MODAL.REMOVE.TITLE'),
+        visible: USER_GROUP_MODAL_TYPE.STATUS,
+        themeColor: 'alert',
+    }); break;
+    case USER_GROUP_MODAL_TYPE.ADD_USERS: userGroupPageStore.updateModalSettings({
+        type,
+        title: i18n.t('IAM.USERGROUP.MODAL.ADD_USERS.TITLE'),
+        visible: USER_GROUP_MODAL_TYPE.ADD_USERS,
+        themeColor: 'primary',
+    }); break;
+    default:
+        break;
+    }
+};
+</script>
+
+<template>
+    <div class="user-group-tab-detail">
+        <p-heading heading-type="sub"
+                   :title="$t('IAM.USERGROUP.MAIN.BASE_INFORMATION')"
+        >
+            <template #extra>
+                <div class="toolbox-wrapper">
+                    <div class="toolbox">
+                        <p-button style-type="tertiary"
+                                  icon-left="ic_edit"
+                                  @click="handleClickButton(USER_GROUP_MODAL_TYPE.EDIT)"
+                        >
+                            {{ $t('IAM.USERGROUP.MAIN.EDIT') }}
+                        </p-button>
+                        <p-button style-type="tertiary"
+                                  icon-left="ic_delete"
+                                  @click="handleClickButton(USER_GROUP_MODAL_TYPE.STATUS)"
+                        >
+                            {{ $t('IAM.USERGROUP.MAIN.REMOVE') }}
+                        </p-button>
+                    </div>
+                </div>
+            </template>
+        </p-heading>
+        <p-definition-table v-if="userGroupPageGetters.selectedUsers.length === 1"
+                            :fields="state.fields"
+                            :data="state.data"
+                            :loading="userGroupPageState.loading"
+                            :skeleton-rows="7"
+                            class="user-group-definition-table"
+                            v-on="$listeners"
+        />
+        <p-definition-table v-else-if="userGroupPageGetters.selectedUsers.length > 1"
+                            :fields="state.fields"
+                            :data="userGroupPageGetters.selectedUsers"
+                            :loading="userGroupPageState.loading"
+                            :skeleton-rows="7"
+                            class="user-group-definition-table"
+                            v-on="$listeners"
+        />
+    </div>
+</template>
+
+<style lang="postcss" scoped>
+.user-group-tab-detail {
+    .toolbox-wrapper {
+        .toolbox {
+            @apply flex;
+            gap: 0.5rem;
+            .button-label {
+                line-height: 1rem;
+            }
+        }
+    }
+
+    /* custom design-system component - p-definition */
+    :deep(.p-definition) {
```

Comment:
nitpick: If a lot of customization is being done on a specific component, it would be good to discuss the changes in the component's properties with the design team.